dashboard/test/models/skill_test.rb

```diff
@@ -0,0 +1,17 @@
+require 'test_helper'
+
+class SkillTest < ActiveSupport::TestCase
+
+  test "can serialize and seed course offerings" do
+    skill = create :skill, key: 'skill-1', description: 'description-1', concept: 'concept-1', evaluation_criteria: 'criteria-1'
+    serialization = skill.serialize
+    previous_skill = skill.freeze
+    skill.destroy!
+    File.stubs(:read).returns(serialization.to_json)
+
+    new_skill_key = Skill.seed_record("config/skills/skill-1.json")
+    new_skill = Skill.find_by(key: new_skill_key)
+    assert_equal previous_skill.attributes.except('id', 'created_at', 'updated_at'),
+      new_skill.attributes.except('id', 'created_at', 'updated_at')
+  end
+end
```

Comment:
nit: please add newline back to end of file, and make sure your editor is set to do this automatically for you. git can get into a weird state when these are missing.

Reply:
arg! sorry this happened again. I updated VS Code and some of my settings stuck and some keep disappearing 🙃